cmd/pro/import_workspace.go

```diff
@@ -45,26 +44,28 @@ func NewImportCmd(globalFlags *flags.GlobalFlags) *cobra.Command {
 
 	importCmd.Flags().StringVar(&cmd.WorkspaceId, "workspace-id", "", "ID of a workspace to import")
 	importCmd.Flags().StringVar(&cmd.WorkspaceUid, "workspace-uid", "", "UID of a workspace to import")
-	importCmd.Flags().StringVar(&cmd.DevPodProUrl, "devpod-pro-url", "", "URL of a DevPod Pro containing the workspace")
 	// optional
 	importCmd.Flags().StringVar(&cmd.WorkspaceContext, "workspace-context", "", "Target context for a workspace")
 	importCmd.Flags().StringArrayVarP(
 		&cmd.WorkspaceOptions, "option", "o", []string{}, "Workspace option in the form KEY=VALUE")
 
 	_ = importCmd.MarkFlagRequired("workspace-id")
 	_ = importCmd.MarkFlagRequired("workspace-uid")
-	_ = importCmd.MarkFlagRequired("devpod-pro-url")
 
 	return importCmd
 }
 
 func (cmd *ImportCmd) Run(ctx context.Context, args []string) error {
+	if len(args) != 1 {
+		return fmt.Errorf("usage: devpod pro import-workspace <workspace-id>")
```

Comment:
this is not the workspace-id and instead the "<instance-host>"

Reply:
yup, devpod-pro-url 😅 thank's for catching it!